pkgs/applications/audio/ardour/ardour5.nix

```
@@ -0,0 +1,165 @@
+{ stdenv
+, fetchgit
+, alsaLib
+, aubio
+, boost
+, cairomm
+, curl
+, doxygen
+, fftwSinglePrec
+, flac
+, glibc
+, glibmm
+, graphviz
+, gtkmm2
+, libjack2
+, libgnomecanvas
+, libgnomecanvasmm
+, liblo
+, libogg
+, librdf_raptor
+, librdf_rasqal
+, libsamplerate
+, libsigcxx
+, libsndfile
+, libusb
```

`libusb` is an alias (which we don't allow in packages). It is aliased to `libusb1`, so that might fix borg's discontent (at least about using aliases):

```suggestion
, libusb1
```

---

ok, done. Thanks!